bundles/org.pitest.pitclipse.ui/src/org/pitest/pitclipse/ui/view/PitView.java

```diff
@@ -16,26 +16,66 @@
 
 package org.pitest.pitclipse.ui.view;
 
+import java.io.File;
+
+import org.eclipse.jface.action.Action;
+import org.eclipse.jface.action.IToolBarManager;
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.SWTError;
 import org.eclipse.swt.browser.Browser;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.MessageBox;
+import org.eclipse.ui.IActionBars;
 import org.eclipse.ui.part.ViewPart;
 
-import java.io.File;
-
 /**
  * A view displaying the HTML report generated by PIT.
  */
 public class PitView extends ViewPart implements SummaryView {
+    public static final String VIEW_ID = "org.pitest.pitclipse.ui.view.PitView";
+    public static final String BLANK_PAGE = "about:blank";
+    public static final String BACK_BUTTON_TEXT = "<";
+    public static final String HOME_BUTTON_TEXT = "Home";
+    public static final String FORWARD_BUTTON_TEXT = ">";
     private Browser browser = null;
+    private String homeUrlString = null;
 
     @Override
     public synchronized void createPartControl(Composite parent) {
         try {
             browser = new Browser(parent, SWT.NONE);
+            resetBrowser();
             browser.addProgressListener(new PitUiUpdatePublisher(browser));
+            // create back button
+            final Action backButtonAction = new Action(BACK_BUTTON_TEXT) {
```

Comment:
Just cosmetic: since we don't store actions in fields anymore, why not avoiding local variables either and just add to the toolbar using an anonymous class?

Reply:
Will do.